translations/extensions/io_xls.py

```diff
@@ -0,0 +1,209 @@
+import datetime
+import glob
+import shutil
+import sys
+import zipfile
+import pandas as pd
+
+from openpyxl import Workbook
+from translations.translator import TranslationRequestGenerator, TranslationResponseProcessor
+from translations.utils import ConfigUtilities, Utilities
+
+
+class Constants:
+    DEFAULT_TRANSL_XLS_PATH = 'translations-xls/'
+    DEFAULT_TRANSL_PKG_NAME = 'translations'
+    WORKING_DIR = 'translations-wrk/'
```

Comment:
Is this supposed to be like a scratch directory that you could theoretically delete afterwards? I think the convention is to either put it in `/tmp` or make it a hidden `.` directory

Reply:
Yeah. I have this in my radar to fix/improve